.devcontainer/plugins/devs-marketplace/plugins/skill-engine/skills/worktree/SKILL.md

```
@@ -0,0 +1,227 @@
+- **Hotfixes** — address an urgent bug without stashing mid-feature work
+- **Agent isolation** — give Claude Code agents their own working directory to prevent file conflicts
+
+---
+
+## Creating Worktrees
+
+### Primary: EnterWorktree Tool (In-Session)
+
+The fastest way to create a worktree during a Claude Code session. Call `EnterWorktree` with a descriptive name:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)